<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItinerary</class>
 <widget class="QWidget" name="AddItinerary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/img/icon.jpg</normaloff>:/new/img/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDateTimeEdit" name="dateTimeEditStart">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>173</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditEnd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRoutes">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAutomobiles">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: #0089D9;
	border: none;
	color: white;
	border-radius: 7px;
}

QPushButton:pressed{
	background: #00598C;}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата/Время начала</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата/Время конца</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрепления</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Маршрут</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
